### ISSUE TITLE
Updating an exisitng model doesn't work

### ISSUE THREAD

**Issue by Globegitter** [kurko/ember-sync#24]
I have the following code to create a new model:

``` js
testModel = this.emberSync.createRecord('test', {
  email: this.get('email'),
  name: this.get('name'),
});
testModel.emberSync.save();
```

This works fine and the model gets saved to the server.

At a later time (after the save has been successfully completed) the user enters something and the model gets updated as follows:

``` js
testModel.set('name', 'anothername');
testModel.set('otherAttribute', 'otherValue');
console.log(testModel.get('isDirty')) //true
testModel.emberSync.save();
```

Ember Inspector shows the new name, otherAttribute and isDirty is true. But no request gets sent to the server.
Removing ember-sync and it works fine.
Any idea what the issue could be?

Edit: Storing offline seems to work though.

**Comment by kurko** [comment]
@Globegitter did you give up on this? I have some time and could take a look at it myself.

**Comment by Globegitter** [comment]
@kurko Yes, we switched to a different solution for that app. I would still love to get this to work because we will most likely revisit ember-sync for a future project.
